src/UKHO.ERPFacade.Common/Models/CloudEvents/S100Event.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System.Diagnostics.CodeAnalysis;
+using Newtonsoft.Json;
+
+namespace UKHO.ERPFacade.Common.Models.CloudEvents.S100
+{
+    [ExcludeFromCodeCoverage]
```

Comment:
Please split this up into multiple files. We should have one file per class or enum definition.

Reply:
Updated as suggested.